bundles/org.openhab.binding.renault/src/main/java/org/openhab/binding/renault/internal/handler/RenaultHandler.java

```diff
@@ -320,10 +336,39 @@ private void updateBattery(MyRenaultHttpSession httpSession) {
                     updateState(CHANNEL_CHARGING_REMAINING_TIME,
                             new QuantityType<Time>(chargingRemainingTime.doubleValue(), MINUTE));
                 }
+                ZonedDateTime batteryStatusUpdated = car.getBatteryStatusUpdated();
+                if (batteryStatusUpdated != null) {
+                    updateState(CHANNEL_BATTERY_STATUS_UPDATED, new DateTimeType(batteryStatusUpdated));
+                }
             } catch (RenaultNotImplementedException e) {
+                logger.warn("Disable cockpit battery update.");
                 car.setDisableBattery(true);
             } catch (RenaultForbiddenException | RenaultUpdateException e) {
+                logger.warn("Error updating battery status.", e);
             }
         }
     }
+
+    private void updateLockStatus(MyRenaultHttpSession httpSession) {
+        if (!car.isDisableLockStatus()) {
+            try {
+                httpSession.getLockStatus(car);
+                updateState(CHANNEL_LOCK_STATUS, new StringType(car.getLockStatus().name()));
+            } catch (RenaultNotImplementedException e) {
+                updateState(CHANNEL_LOCK_STATUS, new StringType(LockStatus.UNKNOWN.name()));
+                logger.warn("Disable lock status update.");
```

Comment:
Under which circumstances will this happen? Perhaps there should be a reason mentioned?

Reply:
Not sure.  It does not work on my car or the testers cars.  However it is documented to work here: https://renault-api.readthedocs.io/en/latest/endpoints.html#lock-status  so it might work for some cars.  If it does not work then the service call is disabled to avoid log SPAM and repeated calls to it.  Next time the thing it initialized all services are enabled again so it might start working one day if the service become available for the car/account.

Reply:
So perhaps "Disabling unsupported lock status update" or something like that?